hack/BUILD.bazel

```diff
@@ -27,6 +47,19 @@ sh_binary(
         GAZELLE,
         KAZEL,
     ],
+    tags = ["manual"],
+)
+
+sh_binary(
+    name = "update-deps-licenses",
+    srcs = ["update-deps-licenses.sh"],
+    args = [
+        "$(location %s)" % GO,
+    ],
+    data = [
+        GO,
+    ],
+    tags = ["manual"],
```

Comment:
`update-deps-licenses` is re-added from [an older commit](https://github.com/cert-manager/cert-manager/blob/3c33140c110631c69214efb3206e5b12b60112ef/hack/BUILD.bazel) but marked manual so it won't be automatically run anywhere.

`update-bazel` and the licenses target become manual